website/docs/plugins/ray_launcher.md

```diff
@@ -64,47 +66,42 @@ defaults:
 config your AWS credentials first, tutorials can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
 
 `ray autoscaler` expects a yaml file to provide specs for the new cluster, which we've schematized in `hydra_ray_launcher.hydra_plugins.hydra_ray_launcher.conf.__init__.RayClusterConf`, 
-we provide a sample yaml file in `conf/hydra/launcher/ray_aws.yaml`. You can either edit the yaml file or override it from the command line to override the 
-default values.
+we provide a sample yaml file in `conf/hydra/launcher/ray_aws.yaml`. You can override the default values in your app config or from command line.
```

Comment:
This is not a sample, it's the plugin defaults that are not in the StructuredConfigs.